changelog.txt

```diff
@@ -1,3 +1,6 @@
+2016-09-19 v0.8.3
+  (fix) enable samba service at wizard
+
```

Comment:
@CvH please add "(add) http://sprunge.us log pastebin function" and "(fix) About text" to the changelog and update the date, then we can merge for inclusion in 7.90.007